runners/flink/build.gradle

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
I recall that `flink_runner.gradle` is imported as a build script to the different flink version builds. Using this file just to do a spotless scan seems fine, but it does split the config and maybe create confusion that nothing is built here.

@mxm @tweise maybe know more about this set up.

Reply:
Is this merely an optimization to avoid running Spotless for each Flink version? If so, probably not worth splitting the config as the overhead for running spotless on the Flink sources multiple times is very low.

Reply:
Besides, I think the spotless task should be part of the actual Flink project, e.g. `runners/flink/1.10` (not `runners/flink`).

Reply:
This is an optimization, but it is also required in order to update the spotless version which is required in order to bump the Gradle version.

In the new version of spotless, all target files must be in the project root which means they can't be added to the buildscripts in the `runners/flink/1.10` subdirectories.
https://github.com/diffplug/spotless/blob/main/plugin-gradle/src/main/java/com/diffplug/gradle/spotless/SpotlessTaskImpl.java#L106

See [this PR](https://github.com/apache/beam/pull/12777) and [this build scan for more info](https://scans.gradle.com/s/mwcogpdbedf7a/failure?anchor=e30&focused-exception-line=0-4-0#1)

Reply:
Thanks for pointers! Just checking, do we spotless-check both `runners/flink/src/**` and `runners/flink/{1.8,1.9,1.10,1.11}/src/**`? It looks like the latter is now covered by the default project settings inherited from BeamModulePlugin.

Reply:
That is correct. I verified in my branch that formatting errors in `runners/flink/src/**` and `runners/flink/1.11/src/**` were caught by spotless.

Reply:
Great, thanks! LGTM then.